vendor/views/vendor_admin.py

```diff
@@ -358,3 +359,35 @@ def post(self, request, *args, **kwargs):
 
         messages.info(request, _("Subscription Renewed"))
         return redirect(request.META.get('HTTP_REFERER', self.success_url))
+
+
+class PaymentWithNoReceiptListView(LoginRequiredMixin, ListView):
+    template_name = "vendor/manage/payment_list.html"
+    model = Payment
+
+    def get_queryset(self):
+        if hasattr(self.request, 'site'):
+            site = self.request.site
+        site = get_site_from_request(self.request)
+        return [payment for payment in Payment.objects.filter(invoice__site=site, success=True) if payment.get_receipt() is None]
+
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['title'] = _("Payments with no Receipts")
+        return context
+
+
+class PaymentWithNoOrderItemsListView(LoginRequiredMixin, ListView):
+    template_name = "vendor/manage/payment_list.html"
+    model = Payment
+
+    def get_queryset(self):
+        if hasattr(self.request, 'site'):
```

Comment:
same here

Reply:
yeah removed that unnecessary check if the function already does it